src/game/objects/general/bridge.c

```diff
@@ -73,20 +76,51 @@ static int32_t Bridge_GetOffset(
     return offset;
 }
 
+static void Bridge_FixEmbedded(int16_t item_num)
+{
+    ITEM_INFO *item;
+    FLOOR_INFO *floor;
+    int32_t x, y, z;
+    int16_t room_number, floor_height, old_anim, old_frame, bridge_height;
+    int16_t *bounds;
+
+    item = &g_Items[item_num];
+    if (item->status != IS_ACTIVE) {
```

Comment:
Consider early exit:

```c
if (item->status == IS_ACTIVE) {
    return;
}
```

Reply:
I removed this check since bridges never have any `ITEM_STATUS`. It was a copy and paste relic from the bat fix.